_includes/specials/policy/en/01-why-mempool.md

```diff
@@ -0,0 +1,79 @@
+<!--
+  300 to 1000 words
+  put title in main newsletter
+  put links in this file
+  for any subheads use h3 (i.e., ###)
+  illustrations welcome (max width 800px)
+  if uncertain about anything, just do what seems best and harding will edit
+-->
+
+Many nodes on the Bitcoin network store unconfirmed transactions in an
+in-memory pool, or "mempool." This cache is an important resource
+for each node and enables the peer-to-peer transaction relay network.
+
+Nodes that participate in transaction relay
+download and validate blocks gradually rather than in spikes.
+Every ~10 minutes when a block is found, nodes without a mempool
+experience a bandwidth spike, followed by a computation-intensive
+period validating each transaction.  On the other hand, nodes with a
+mempool have typically already seen all of the block transactions and
+store them in their mempools. With compact block relay (fixme: 152
+link), these nodes just download a block header along with shortids,
+and then reconstruct the block using transactions in their mempools.
+The amount of data used to relay compact blocks is tiny compared to
+the size of the block (fixme: stats?). Validating the transactions is
+also much faster: the node has already verified (and cached)
+signatures and scripts, calculated the timelock requirements, and
+loaded relevant UTXOs from disk if necessary. The node can also
+forward the block onto its other peers promptly, dramatically
+increasing network-wide block propagation speed and thus reducing the
+risk of stale blocks.
+
+Mempools can also be used to build an independent fee estimator. The
+market for block space is a fee-based auction, and keeping a mempool
+allows users to have a better sense of what others are bidding and
+what bids have been successful in the past.
+
+However, there is no such thing as "the mempool" - each node may
+receive different transactions. Submitting a transaction to one node
+does not necessarily mean that it has made its way to miners. Some
+users find this uncertainty frustrating, and wonder, "why don't we
+just submit transactions directly to miners?"
+
+Consider a Bitcoin network in which all transactions are sent directly
+from users to miners. One could censor and surveil financial activity
+by requiring the small number of entities to log the IP addresses
+corresponding to each transaction, and refuse to accept any
+transactions matching a particular pattern. This type of Bitcoin may
+be more convenient at times, but would be missing a few of Bitcoin's
+most valued properties.
```

Comment:
I think this is a fine description of one of the benefits of a decentralized relay network, and I don't think it needs to be changed, but I wanted to note that I've always felt that the strongest argument for an independent relay network is to avoid naturally privileging lager miners over smaller miners.  E.g., [this argument](https://gist.github.com/harding/7c6c5215a0aec8e0b8b186889def01d8)

Reply:
Really good point, tried to incorporate this a bit.